extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -66,13 +69,31 @@ export const RTC_ERROR_ENUM = {
  * @private
  */
 function buildErrorResponse_(error, callout, opt_rtcTime, opt_log) {
+  callout = getCalloutParam(callout);
   if (opt_log) {
     dev().warn(TAG, `Dropping RTC Callout to ${callout} due to ${error}`);
   }
   return Promise.resolve(/**@type {rtcResponseDef} */(
     {error, callout, rtcTime: opt_rtcTime || 0}));
 }
 
+/**
+ * Converts a URL into its corresponding shortened callout string, or if
+ * passed a vendor name, just returns the vendor. For instance, if we are
+ * passed "https://example.com/example.php?foo=a&bar=b, then we return
+ * example.com/example.php
+ * and if we are passed "exampleVendor" then we return "exampleVendor"
+ * @param {string} callout
+ * @return {string}
+ */
+export function getCalloutParam(callout) {
+  try {
+    const parsedUrl = parseUrl(callout);
+    callout = (parsedUrl.hostname + parsedUrl.pathname).substr(0, 50);
+  } catch (e) {}
```

Comment:
What sort of errors are we trying to catch here?

We should generally be doc'ing what sort of errors lead to us just returning the input unmodified.

Reply:
done